extensions/extensions-docker/pom.xml

```diff
@@ -99,20 +106,24 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>org.technologybrewery.orphedomos</groupId>
-                <artifactId>orphedomos-maven-plugin</artifactId>
-                <configuration>
-                    <buildArgs>
-                        <!-- allows arch-specific builds to reference the exact base image, might not be needed once we do multi-arch builds + manifest in Orphedomos -->
-                        <VERSION_AISSEMBLE>${dockerImageVersion}</VERSION_AISSEMBLE>
-                        <DELTA_HIVE_CONNECTOR_VERSION>${version.delta.hive.connector}</DELTA_HIVE_CONNECTOR_VERSION>
-                    </buildArgs>
-                    <aliases>
-                        <!-- Allows downstream projects/modules to use the locally built image -->
-                        <!-- It may make sense to have Orphedomos build the image with the repo included, which would remove the need for this -->
-                        <alias>${docker.baseline.repo.id}/${dockerImageName}:${dockerImageVersion}</alias>
```

Comment:
I: Images now include the `${docker.baseline.repo.id}` by default so no need for the alias in fabric8.